ssr-testing/pages/radio-group.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { RadioGroup, RadioGroupItem, RadioGroupIndicator } from '@radix-ui/react-radio-group';
```

Comment:
We're able to import the packages like this, and they import the source and everything rebuilds and hot reloads when making changes.